operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -266,6 +266,10 @@ type PerNodeSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3,displayName="Node Scanning Settings"
 	NodeInventory *ContainerSpec `json:"nodeInventory,omitempty"`
 
+	// Settings for the SFA container, which is responsible for file activity monitoring on the Node.
```

Comment:
Please
- expand the SFA acronym here and in the `displayName`
- use `order=4` and renumber the fields below

The field name is `SFA` but the schema field name is `fact`. I think it would be more straightforward if these matched 🤔